src/main/java/lotto/Lotto.java

```diff
@@ -1,20 +1,167 @@
 package lotto;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.Set;
+
+import static PrintEnums.ErrorMessages.OUTPUT_ERROR;
+import static PrintEnums.ErrorMessages.OUTPUT_ERROR_THOUSAND;
+import static PrintEnums.MagicNumbers.*;
+
 
 public class Lotto {
     private final List<Integer> numbers;
+    //numbers 수정X
+    //필드추가 ㅇㅋ
 
+    //생성자로 초기화 List 넣자마자 작동 로또넘버를 입력받음.
     public Lotto(List<Integer> numbers) {
+        validateDuplicate(numbers);
         validate(numbers);
+        validateLottoNumber(numbers);
```

Comment:
```suggestion
        validate(numbers);
        
        public void validate(List<Integer> numbers) {
            validateDuplicate(numbers);
            validateLottoNumber(numbers);
        }
```
세개의 validate 메서드가 다 같은 파라미터를 받고 있습니다.
예를 들어서 validate(numbers) 를 두고 validate 메서드 안에서 저 세개의 유효성 검사를 두면 코드상 더 깔끔해질거 같습니다!